spec/mongoid/clients/sessions_spec.rb

```diff
@@ -0,0 +1,297 @@
+require "spec_helper"
```

Comment:
Should there be any tests that use multiple threads?

Reply:
Sure, that's a good idea. I'll add a test to make sure a session is not accessible from another thread.